### DISCUSSION TITLE
[FEATURE]  QR Payment for ASEAN, QRIS

### DISCUSSION THREAD

**Discussion by KagChi** [juspay/hyperswitch#2824] [Ideas / Feature Requests]
### Feature Description

User in ASEAN can pay when they scan the qr payment

### Possible Implementation

Can either use https://qris.online/homepage/ or https://midtrans.com/en

### Have you spent some time checking if this feature request has been raised before?

- [X] I checked and didn't find a similar issue

### Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/juspay/hyperswitch/blob/main/docs/CONTRIBUTING.md)

### Are you willing to submit a PR?

No, I don't have time to work on this right now

**Comment by chez14** [comment]
Adding my two cents, QRIS.online, QRIS.id, or interactiveQRIS [is widely known](https://twitter.com/search?q=qris.id) to be problematic in the community, so please proceed with caution.

Alternatively, here are other options for the processors of QRIS:
- Xendit: https://www.xendit.co/en/
- Doku: https://www.doku.com/en-US

Or see the list here:
- https://www.aspi-indonesia.or.id/standar-dan-layanan/qris/

**Comment by ariaseta** [comment]
Yes, prefer Midtrans and Xendit for first
